content/en/docs/tutorials/stateful-application/cassandra.md

```diff
@@ -1,85 +1,72 @@
 ---
-title: "Example: Deploying Cassandra with Stateful Sets"
+title: "Example: Deploying Cassandra with a StatefulSet"
 reviewers:
 - ahmetb
 content_template: templates/tutorial
 weight: 30
 ---
 
 {{% capture overview %}}
-This tutorial shows you how to develop a native cloud [Cassandra](http://cassandra.apache.org/) deployment on Kubernetes. In this example, a custom Cassandra *SeedProvider* enables Cassandra to discover new Cassandra nodes as they join the cluster.
+This tutorial shows you how to run [Apache Cassandra](http://cassandra.apache.org/) on Kubernetes. Cassandra, a database, needs persistent storage to provide data durability (application _state_). In this example, a custom Cassandra seed provider lets the database discover new Cassandra instances as they join the Cassandra cluster.
 
-*StatefulSets* make it easier to deploy stateful applications within a clustered environment. For more information on the features used in this tutorial, see the [*StatefulSet*](/docs/concepts/workloads/controllers/statefulset/) documentation.
-
-**Cassandra on Docker**
-
-The *Pods* in this tutorial use the [`gcr.io/google-samples/cassandra:v13`](https://github.com/kubernetes/examples/blob/master/cassandra/image/Dockerfile)
-image from Google's [container registry](https://cloud.google.com/container-registry/docs/).
-The Docker image above is based on [debian-base](https://github.com/kubernetes/kubernetes/tree/master/build/debian-base)
-and includes OpenJDK 8.
-
-This image includes a standard Cassandra installation from the Apache Debian repo.
-By using environment variables you can change values that are inserted into `cassandra.yaml`.
-
-| ENV VAR       | DEFAULT VALUE  |
-| ------------- |:-------------: |
-| `CASSANDRA_CLUSTER_NAME` | `'Test Cluster'`  |
-| `CASSANDRA_NUM_TOKENS`  | `32`               |
-| `CASSANDRA_RPC_ADDRESS` | `0.0.0.0`          |
+*StatefulSets* make it easier to deploy stateful applications into your Kubernetes cluster. For more information on the features used in this tutorial, see [StatefulSet](/docs/concepts/workloads/controllers/statefulset/).
 
+{{< note >}}
+Cassandra and Kubernetes both use the term _node_ to mean a member of a cluster. In this
+tutorial, the Pods that belong to the StatefulSet are Cassandra nodes and are members
+of the Cassandra cluster (called a _ring_). When those Pods run in your Kubernetes cluster,
+the Kubernetes control plane schedules those Pods onto Kubernetes
+{{< glossary_tooltip text="Nodes" term_id="node" >}}.
```

Comment:
nit: Still, I think you are referring to Kubernetes nodes (machines)?

Reply:
I'm not sure I understand this question. I can confirm that `{{< glossary_tooltip text="Nodes" term_id="node" >}}` refers to Kubernetes nodes.